$MirOS: ports/lang/pcc/patches/patch-configure_ac,v 1.1.201.1 2008/05/02 18:29:52 tg Exp $

	• fix stupid build system ignoring our CFLAGS
	• fix usage of autoconf
	• brand and anonymise

--- configure.ac.orig	Thu Jan  1 00:00:00 1970
+++ configure.ac	Thu Jul 17 21:02:19 2008
@@ -7,6 +7,9 @@ AC_CONFIG_HEADER([config.h])
 
 AC_CANONICAL_TARGET
 
+ORIG_CFLAGS=$CFLAGS
+AC_SUBST([ORIG_CFLAGS])
+
 abi=unknown
 endian=little
 tls=no
@@ -157,13 +160,13 @@ else
 fi
 
 case "$abi" in
-	elf*)		AC_DEFINE(ELFABI, [], "Using ELF ABI") ;;
-	aout)		AC_DEFINE(AOUTABI, [], "Using a.out ABI") ;;
-	macho)		AC_DEFINE(MACHOABI, [], "Using Mach-O ABI") ;;
-	coff)		AC_DEFINE(COFFABI, [], "Using COFF ABI") ;;
-	ecoff)		AC_DEFINE(ECOFFABI, [], "Using ECOFF ABI") ;;
-	pecoff)		AC_DEFINE(PECOFFABI, [], "Using PE/COFF ABI") ;;
-	classic68k)	AC_DEFINE(CLASSIC68K, [], "Using Classic 68k ABI") ;;
+	elf*)		AC_DEFINE(ELFABI, [], [Using ELF ABI]) ;;
+	aout)		AC_DEFINE(AOUTABI, [], [Using a.out ABI]) ;;
+	macho)		AC_DEFINE(MACHOABI, [], [Using Mach-O ABI]) ;;
+	coff)		AC_DEFINE(COFFABI, [], [Using COFF ABI]) ;;
+	ecoff)		AC_DEFINE(ECOFFABI, [], [Using ECOFF ABI]) ;;
+	pecoff)		AC_DEFINE(PECOFFABI, [], [Using PE/COFF ABI]) ;;
+	classic68k)	AC_DEFINE(CLASSIC68K, [], [Using Classic 68k ABI]) ;;
 esac
 
 # Specify alternate assembler, linker, include and lib paths
@@ -344,7 +347,7 @@ AC_SUBST(PACKAGE_VERSION)
 pcc_major=`echo $PACKAGE_VERSION | awk -F. '{print $1}'`
 pcc_minor=`echo $PACKAGE_VERSION | awk -F. '{print $2}'`
 pcc_minorminor=`echo $PACKAGE_VERSION | awk -F. '{print $3}'`
-versstr="\"$PACKAGE_STRING for $target, $USER@`hostname` `date`\""
+versstr="\"$PACKAGE_STRING for $target, ${MPVERSION-USER@`hostname` `date`}\""
 
 AC_DEFINE_UNQUOTED(TARGOS, $targos, [Target OS])
 AC_DEFINE_UNQUOTED(PCC_MAJOR, $pcc_major, [Major version no])
